extensions/wikia/CreateNewPageFlow/scripts/flowTracking.js

```diff
@@ -0,0 +1,34 @@
+define('flowTracking', ['wikia.cookies', 'wikia.tracker', 'mw'], function (cookies, tracker, mw) {
+	var cookieName = 'CNPFlowLabel',
```

Comment:
Per comment regarding
https://github.com/Wikia/app/pull/11504/files#diff-bede951fb0e48db15998e5e466d5b2ceR3071
does it need to be CNP specific?